collections/nemo_nlp/nemo_nlp/data/datasets/utils.py

```diff
@@ -547,6 +547,134 @@ def merge(data_dir, subdirs, dataset_name, modes=['train', 'test']):
     return outfold, none_slot
 
 
+def get_intent_query_files(path):
+    files = []
+    # r=root, d=directories, f = files
+    for r, d, f in os.walk(path):
+        for file in f:
```

Comment:
file is a keyword. use file_ for filename

Reply:
It is not a keyword but a built-in function - https://stackoverflow.com/questions/24942358/is-file-a-keyword-in-python
Also, there are usage of 'file' as a variable name in other parts of the code in this utils.py file, which is not written by me. We could keep them as it since they won't be an issue in python 3 or we should change all of them for consistency. let me know. :)

Reply:
good point. if it's not a problem in python3 then we can keep it as is!